### ISSUE TITLE
[Crash-1.12.2]Crash while starting up

### ISSUE THREAD

**Issue by Tollainmear** [turbodiesel4598/NuclearCraft#305]
Crashed while `getting property PropertyEnum` on server startted up

[crash-2018-05-02_16.02.40-server.txt](https://github.com/turbodiesel4598/NuclearCraft/files/1966409/crash-2018-05-02_16.02.40-server.txt)

Using NC-2.10m   
Forge 2655

That issue make server can't run, so reply soon if possible plz 😅

**Comment by turbodiesel4598** [comment]
I have a strong suspicion that this is something to with Sponge - it is known to cause these sorts of issues... will go talk to the devs :)

**Comment by Tollainmear** [comment]
Thank you for reply, I'll starting the server without spongeforge for a test soon

**Comment by Tollainmear** [comment]
BTW, I have dowload the world map from server then open it on client-side(That means running without spongeforge), and client would crashed with the same kind of crash report.
I have test to run the server with spongeforge uninstalled just now, and it crashed agin :sweat_smile:

**Comment by turbodiesel4598** [comment]
Ok, unfortunately I don't know the details of your setup, but if you [post a bug report here](https://github.com/SpongePowered/SpongeForge/issues) there will hopefully be a solution.

**Comment by Tollainmear** [comment]
OK, should take more test before that, I would leave a link here after posting issue for sponge forge, Thanks for helping.
Hope NC being more and more popular ;)

**Comment by turbodiesel4598** [comment]
That's applied energistics - loos like some part of it was looking for an ME controller but unexpectedly found a NuclearCraft fluid extractor.

**Comment by Tollainmear** [comment]
I have back to check each mod, and then I found that [log.txt](https://github.com/turbodiesel4598/NuclearCraft/files/1967985/log.txt)

So, I'm more confused now :flushed:.

what should I do now...........

**Comment by Tollainmear** [comment]
Finally, I decided to post a issue to [AE2 issue tracker](https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/3477), I'm sorry about haven't accept your suggestion, I just try to find some relationship between NC and SpongeForge, but even if I deleted the spongeforge, that crash still happened with the same crash-report 😅,but it is obvious that the crash might caused by AE2.

however, I still identify with your opinion, thank you for your help.
That would be great if we make this issue kept opening  until we got more info.
